Inspicio/wwwroot/css/site.css

```diff
@@ -116,6 +116,8 @@ body {
     font-size: 18px;
     align-content: center;
     table-layout: fixed;
+    margin-top:50px;

```

Comment:
Does not make any difference, but could just use `margin:50px 0;` . First number for topb/bottom, second for sides.